client/my-sites/plan-features-2023-grid/index.jsx

```diff
@@ -258,6 +263,52 @@ export class PlanFeatures2023Grid extends Component {
 			<PlanFeatures2023GridFeatures planProperties={ planProperties } domainName={ domainName } />
 		);
 	}
+
+	getFeatureToStorageMap( storageFeature ) {
+		switch ( storageFeature ) {
+			case FEATURE_1GB_STORAGE:
+				return translate( '1GB' );
+			case FEATURE_6GB_STORAGE:
+				return translate( '6GB' );
+			case FEATURE_13GB_STORAGE:
+				return translate( '13GB' );
+			case FEATURE_200GB_STORAGE:
+				return translate( '200GB' );
```

Comment:
Minor: In the design, the text is `200 GB` (there's a space between 200 & GB)
<img width="320" alt="image" src="https://user-images.githubusercontent.com/5436027/209946420-ce9f2381-f4d4-4a10-a9fe-383051438c96.png">

Reply:
You are right about the design, but in the product we seem to be using both representations - `X GB` and `XGB`, e.g [this line](https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/plans/features-list.js#L1324). I'll verify with the designer on which approach to take and I think we can have this as a follow-up fix.